storm-core/src/jvm/backtype/storm/serialization/DefaultSerializationDelegate.java

```diff
@@ -17,11 +17,7 @@
  */
 package backtype.storm.serialization;
```

Comment:
This is really minor, but it might also be nice to deprecate this, and migrate the core deserialization over to Utils, so we are not calling a deprecated class.